docs/source/entities-advanced.mdx

```diff
@@ -36,6 +37,24 @@ type Organization {
 }
 ```
 
+<Note>
+
+Although nested fields are most commonly used in compound keys, you can also use a nested field as a single `@key` field.
+For example, the previous `User` entity `@key` doesn't make much sense without including the user `id`:
```

Comment:
IIUC, this example shows why nested fields are usually part of a compound key. But from the first sentnece, the purpose of this note is to say that a nested key can be the only field of a simple key? If so, is there a practical example of a simple key with a nested field?

Reply:
> If so, is there a practical example of a simple key with a nested field?

I was struggling to think of one 😅, let me ask the SA team...